torch/nn/utils/rnn.py

```diff
@@ -158,7 +176,21 @@ def pad_packed_sequence(sequence, batch_first=False, padding_value=0):
 
     if batch_first:
         output = output.transpose(0, 1)
-    return output, lengths
+    return output, Variable(torch.LongTensor(lengths))
```

Comment:
This needs a comment saying that this Variable doesn't actually have any history, it is purely here to make ONNX export easier.

Reply:
++